ansible-101/Dockerfile.ssh

```diff
@@ -1,4 +1,4 @@
-FROM debian:stable-slim
+FROM debian:stable
```

Comment:
Mi risulta

```
docker run --rm  debian:stable bash -c 'apt-get update && apt-get install -y python'
Get:1 http://deb.debian.org/debian stable InRelease [147 kB]
Get:2 http://deb.debian.org/debian-security stable-security InRelease [48.0 kB]
Get:3 http://deb.debian.org/debian stable-updates InRelease [52.1 kB]
Get:4 http://deb.debian.org/debian stable/main amd64 Packages [8904 kB]
Get:5 http://deb.debian.org/debian-security stable-security/main amd64 Packages [11.5 kB]
Fetched 9163 kB in 2s (5385 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
Package python is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  python-is-python3 2to3

E: Package 'python' has no installation candidate
```